### PR TITLE
MOS overview

### PR DESCRIPTION
**Pull request by brittBarak**
## Description

A short explanation on MOS- a measurement for voice or video quality

### CODE REVIEW

File: _documentation/stitch/concepts/mos.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+The overall quality of a voice or video stream is often depicted through a measure called [*Mean Opinion Score (MOS)*](https://en.wikipedia.org/wiki/Mean_opinion_score).
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale: 
+> 5 - Excellent, 4 - Good, 3 - Fair, 2 - Poor, 1 - Bad.
+
+Frequently, MOS higher than 4 reflects a satisfying quality. For MOS higher than 4.3, the human senses cannot usually perceive an increase in quality. On the lower side, when MOS is lower than 3.5, commonly the quality isn’t sufficient, especially for communication purposes.
+
+
+### How is MOS calculated?
+The MOS is calculated using an agreed upon formula, which utilizes multiple features of the steam passed over the network. Among those features are: jitter, packet loss, latency and network state.
```
[the remark this answers is not in the document]
Comment (brittBarak) [reply _documentation/stitch/concepts/mos.md]: I'm trying to get an official link to it, but it's fairly multi-variables calculation.  a part of the actual algo, is there anything else you'd like to see?

[the remark this answers is not in the document]
Comment (AlexLakatos) [reply _documentation/stitch/concepts/mos.md]: oh, the wikipedia MOS page formula is about human subjects, or subjective readings. nothing like that.
I wanted the page to talk a bit about the actual formula Stitch uses to calculate MOS

[the remark this answers is not in the document]
Comment (brittBarak) [reply _documentation/stitch/concepts/mos.md]: Yes, that's what I meant. there's an agreed upon formula for calculating MOS, which I'm just trying to get an official link to, so that it's clear that we didn't invent it or something. 

you can find it here for now:
https://nexmoinc.atlassian.net/wiki/spaces/STCH/pages/486146178/Voice+Quality+Metrics+Tech+Spec?preview=/486146178/532480083/calcmos_h%20

File: _documentation/stitch/concepts/mos.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+In the past, a group of individuals was subjectively rating the quality of the experience. An arithmetic mean was calculated and gave a final score which represented the quality score. Later on, these scores were the base for the MOS formula, which now provides an objective score, which mostly represents the network quality.
```
[the remark this answers is not in the document]
Comment (ericgiannini) [reply _documentation/stitch/concepts/mos.md]: Do you have the formula for which the MOS is calculated? It is mentioned in line 16 but not given.

File: _documentation/stitch/concepts/mos.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Measuring Voice Or Video Quality
+---
+
+# Measuring Voice Or Video Quality
+
+The overall quality of a voice or video stream is often depicted through a measure called [*Mean Opinion Score (MOS)*](https://en.wikipedia.org/wiki/Mean_opinion_score).
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale:
```
[the remark this answers is not in the document]
Comment (ericgiannini) [reply _documentation/stitch/concepts/mos.md]: Will our MOS scores have floats or will ours have whole numbers?

File: _documentation/stitch/concepts/mos.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (brittBarak) [reply _documentation/stitch/concepts/mos.md]: Adding a link from Guy. Don't think copying a mathematical formula is helpful here..

File: _documentation/stitch/concepts/mos.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: Measuring Voice Or Video Quality
+---
+
+# Measuring Voice Or Video Quality
+
+The overall quality of a voice or video stream is often depicted through a metric called [*Mean Opinion Score (MOS)*](https://en.wikipedia.org/wiki/Mean_opinion_score).
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale: 
+> 5 - Excellent, 4 - Good, 3 - Fair, 2 - Poor, 1 - Bad.
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: +1 on having this as a list e.g.

1. Bad
2. Poor
3. Fair
4. Good
5. Excellent

File: _documentation/stitch/concepts/mos.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+title: Measuring Voice Or Video Quality
+---
+
+# Measuring Voice Or Video Quality
+
+The overall quality of a voice or video stream is often depicted through a metric called [*Mean Opinion Score (MOS)*](https://en.wikipedia.org/wiki/Mean_opinion_score).
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale:
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: I had to Google what a rational number was to make sure that I'd remembered correctly. Can we describe it any other way?

File: _documentation/stitch/concepts/mos.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+# Measuring Voice Or Video Quality
+
+The overall quality of a voice or video stream is often depicted through a metric called [*Mean Opinion Score (MOS)*](https://en.wikipedia.org/wiki/Mean_opinion_score).
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale: 
+> 5 - Excellent, 4 - Good, 3 - Fair, 2 - Poor, 1 - Bad.
+
+Frequently, MOS higher than 4 reflects a satisfying quality. For MOS higher than 4.3, the human senses cannot usually perceive an increase in quality. On the lower side, when MOS is lower than 3.5, commonly the quality isn’t sufficient, especially for communication purposes.
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: Do we need to tell people that we can't perceive any difference above 4.3? We can reword this to be more straightforward if it's not included

> A MOS of 4 means that the quality of the audio or video was satisfactory. If the MOS is lower than 3.5, it's a strong indication that the quality if not sufficient (especially for communication purposes).

File: _documentation/stitch/concepts/mos.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### How is MOS calculated?
+
+The MOS is calculated using an [agreed upon formula](https://docs.telcobridges.com/tbwiki/MOS), which utilizes multiple features of the data steam passed over the network. Among those features, the ones with the most significant impact are: 
+* Packet loss - the amount of data packets that never made it from the sender to the receiver
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: Needs a line break before this to render as a list

File: _documentation/stitch/concepts/mos.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+It is common that MOS is represented as a rational number between 1 and 5, with the following scale: 
+> 5 - Excellent, 4 - Good, 3 - Fair, 2 - Poor, 1 - Bad.
+
+Frequently, MOS higher than 4 reflects a satisfying quality. For MOS higher than 4.3, the human senses cannot usually perceive an increase in quality. On the lower side, when MOS is lower than 3.5, commonly the quality isn’t sufficient, especially for communication purposes.
+
+
+### How is MOS calculated?
+
+The MOS is calculated using an [agreed upon formula](https://docs.telcobridges.com/tbwiki/MOS), which utilizes multiple features of the data steam passed over the network. Among those features, the ones with the most significant impact are:
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: Who agreed upon this formula? That link looks like a single vendor to me

File: _documentation/stitch/concepts/mos.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### How is MOS calculated?
+
+The MOS is calculated using an [agreed upon formula](https://docs.telcobridges.com/tbwiki/MOS), which utilizes multiple features of the data steam passed over the network. Among those features, the ones with the most significant impact are: 
+* Packet loss - the amount of data packets that never made it from the sender to the receiver
+* Latency - the time it takes a data packet to get from the sender to the receiver, and back to the sender
+* Jitter - the difference between the latency of one data flow to the other. In other words, how delayed was a received data packet, comparing to the previous received data packet
+
+An R-value is calculated based on those factors and is later used to calculate the final MOS, by this well unknown formula:
```
Comment (mheap) [inline _documentation/stitch/concepts/mos.md]: Well unknown formula?

File: _documentation/stitch/concepts/mos.md (hunk 8 above)
Comment (gmininberg) [reply _documentation/stitch/concepts/mos.md]: this is the formula we use in VBC rtpProxy. you also use it on MB you can ask Neil and Dragos about it.

Comment (brittBarak) [reply _documentation/stitch/concepts/mos.md]: @mheap  are we good with the above?

Comment (mheap) [reply _documentation/stitch/concepts/mos.md]: I'd drop the `agreed upon` part and just state that we use that formula for MOS

File: _documentation/stitch/concepts/mos.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+* Packet loss - the amount of data packets that never made it from the sender to the receiver
+
+* Latency - the time it takes a data packet to get from the sender to the receiver, and back to the sender
+
+* Jitter - the difference between the latency of one data flow to the other. In other words, how delayed was a received data packet, comparing to the previous received data packet
+
+
+An R-value is calculated based on those factors and is later used to calculate the final MOS, by this well known formula:  
+> MOS = 1 + 0.035R + ((R - 60) * (100 – R) * 0.000007R)
+
```
[the remark this answers is not in the document]
Comment (brittBarak) [reply _documentation/stitch/concepts/mos.md]: like I did at first and though you didnt want... k I'm adding

File: _documentation/stitch/concepts/mos.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+
+### What can a MOS can be used for
```
Comment (lornajane) [inline _documentation/stitch/concepts/mos.md]: Maybe "Uses for MOS Mean Opinion Score" ?  Definitely there is at least one "can" too many in this title.

**Comment by ChrisGuzman** [comment]
@brittBarak I think we're good to merge this if you are
